internal/convert.go

```diff
@@ -147,8 +147,6 @@ type statementStat struct {
 // Stores the audit information of conversion.
 // Elements that do not affect the migration functionality but are relevant for the migration metadata.
 type Audit struct {
-	ToSpannerFkIdx           map[string]FkeyAndIdxs                 // Maps from source-DB table name to Spanner names for table name, foreign key and indexes.
```

Comment:
We should remove the definitions of these from convert.go and conv.ts as well

Reply:
Removed. Please review it.